cirq/protocols/commutes_protocol_test.py

```diff
@@ -15,6 +15,7 @@
 import numpy as np
 import pytest
 import sympy
+import copy
```

Comment:
You are not using this import. Remove it.

Reply:
Roger that